tests/Feature/RankingTest.php

```diff
@@ -36,6 +38,7 @@ protected function setUp(): void
         $this->quiz = $seeder->quiz;
         $seeder->createSubmissionForUser($this->user1, 2);
         $seeder->createSubmissionForUser($this->user2, 3);
+        $this->quiz1 = Quiz::factory()->locked()->create();
```

Comment:
You don't need set up `quiz1` for every tests in this files. Create where you want to use this.
By the way, maybe more semantic name for that. What does it mean `quiz 1` ?